src/FrequencyMapAgent.hpp

```diff
@@ -79,6 +79,11 @@ namespace geopm
             static std::unique_ptr<Agent> make_plugin(void);
             static std::vector<std::string> policy_names(void);
             static std::vector<std::string> sample_names(void);
+            void set_maps(const std::map<uint64_t, double>& hash_freq_map,
+                          const std::set<uint64_t>& default_freq_hash) {
+                m_hash_freq_map = hash_freq_map;
+                m_default_freq_hash = default_freq_hash;
+            }
```

Comment:
I think it would be better to test this through our integration tests than to introduce a new class method that is only used in testing.

Typically when we're testing the behavior of agents, this is done through an integration test.

This would also illustrate the indentation issues I commented on.  You won't see that in a unit test.